README.md

```diff
@@ -1 +1,199 @@
-# Full-Stack-Capstone
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+
+  <h1><b>Land Rover E-Commerce</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Back-End React](#backend)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [🙏 Credits](#credit)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Land Rover E-Commerce <a name="about-project"></a>
+
+**Land Rover E-Commerce** "The Final Capstone Project is based on an E-commerce app to make a reservation to try Land Rover vehicle."
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="#">HTML</a></li>
+  </ul>
+</details>
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="#">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>ReactJS</summary>
+  <ul>
+    <li><a href="#">ReactJS</a></li>
+  </ul>
+</details>
+<details>
+<summary>Github</summary>
+  <ul>
+    <li><a href="#">Github</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+### Back End React <a name="#backend"></a>
+
+- [Back-End React](https://github.com/VelzckC0D3/land-rover-back-end.git) 
+
+### Key Features <a name="key-features"></a>
+
+- **[Setup with vite]**
+- **[Setup linters for React/Redux]**
+- **[Setup React TailwindCSS]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+Node.js
+Visual Studio Code
+Browser
+
+### Setup
+
+git clone https://github.com/VelzckC0D3/Land-Rover-E-Commerce.git
+
+cd Land-Rover-E-Commerce
+
+### Install
+
+npm install
+
+### Usage
+
+npm run start
```

Comment:
- [ ] This command won't work to start the project:

![image](https://github.com/VelzckC0D3/Land-Rover-E-Commerce/assets/57421823/1139f927-28ea-40e5-84bf-735190d1fb09)

As it's stated in `package.json`, The correct command is `npm run dev`.   Kindly update your instructions.

That way, your users will be able to run your project. 👍